examples/statistics/confidence_ellipse.py

```diff
@@ -16,6 +16,7 @@
 pearson correlation coefficients and ones) is particularly easy to handle.
 """
 
+
 #############################################################################
 #
 # The plotting function itself
```

Comment:
You can't mix the comment types w/o some code between.  Suggest putting the imports at line 19